modules/installation-azure-dedicated-disks.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_azure/ipi/installing-azure-customizations.adoc
+// * installing/installing_azure/ipi/installing-azure-network-customizations.adoc
+// * installing/installing_azure/ipi/installing-restricted-networks-azure-installer-provisioned.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="installation-azure-dedicated-disks_{context}"]
+= Configuring a dedicated disk for etcd
+
+You can install your {product-title} cluster on {azure-first} with a dedicated data disk for `etcd`. This configuration attaches a separate managed disk to each control plane node and uses it only for `etcd` data, which can improve cluster performance and stability.
+
+:FeatureName: Dedicated disk for etcd
+
+include::snippets/technology-preview.adoc[]
+
+.Prerequisites
+
+* You have created an `install-config.yaml` file.
+
+.Procedure
+
+* To configure a dedicated `etcd` disk, edit the `install-config.yaml` file and add the `diskSetup` and `dataDisks` parameters to the `controlPlane` stanza:
++
+[source,yaml]
+----
+controlPlane:
+  architecture: amd64
+  hyperthreading: Enabled
+  name: master
+  platform:
+    azure:
+      type: Standard_D4s_v5
+      dataDisks:
+      - nameSuffix: etcddisk <1>
+        cachingType: None <2>
+        diskSizeGB: 20 <3>
+        lun: 0 <4>
+  diskSetup:
+  - type: etcd <5>
+    etcd:
+      platformDiskID: etcddisk <6>
```

Comment:
https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#using-asciidoc-callouts-in-the-yaml

> Comment out the callout in the YAML file to ensure that file can still be parsed as valid YAML....